server/odc-service/src/main/java/com/oceanbase/odc/service/dlm/DLMTableStructureSynchronizer.java

```diff
@@ -67,25 +69,35 @@ public static void sync(ConnectionConfig sourceConnectionConfig, ConnectionConfi
         DBStructureComparisonConfig targetConfig = initDBStructureComparisonConfig(
                 targetConnectionConfig, tableNames);
         DefaultDBStructureComparator comparator = new DefaultDBStructureComparator();
-        List<DBObjectComparisonResult> results = comparator.compare(sourceConfig, targetConfig).stream()
-                .peek(o -> o.setSubDBObjectComparisonResult(o.getSubDBObjectComparisonResult().stream()
-                        .filter(sb -> targetType.contains(sb.getDbObjectType())).collect(
-                                Collectors.toList())))
-                .collect(Collectors.toList());
-        try (Connection conn = targetConfig.getDataSource().getConnection()) {
-            for (DBObjectComparisonResult result : results) {
-                String changeSqlScript =
-                        result.toEntity(1L, targetConnectionConfig.getDialectType()).getChangeSqlScript();
-                if (StringUtils.isNotEmpty(changeSqlScript)) {
-                    log.info("Start to sync target table structure,sqls={}", changeSqlScript);
-                    conn.prepareStatement(changeSqlScript).execute();
-                    log.info("Sync table structure success.");
-                } else {
-                    log.info("Table structure comparison has finished,no action is necessary.");
+        DBObjectComparisonResult result = comparator.compare(sourceConfig, targetConfig).get(0);
```

Comment:
Will there occurs that the compare result has both `ONLY_IN_SOURCE` and `INCONSISTENT`?

Reply:
No, the INCONSISTENT enum value indicates that it exists, but is not consistent.